### ISSUE TITLE
The issue for each problematic user is added twice during one audit

### ISSUE THREAD

**Issue by epoberezkin** [guardian/gu-who#63]

**Comment by kaelig** [comment]
I have the same issue, looks like GitHub is sending 2 payloads in a 5-second interval and this is making the bot create 2 issues.

**Comment by rtyley** [comment]
Yup, I see this too - it is annoying. In Prout I added some [dog-piling code](https://github.com/guardian/prout/blob/cedbcb914/app/lib/Dogpile.scala#L8-L19) to stop this kind of issue, but haven't done the equivalent for _gu:who_ yet.

**Comment by epoberezkin** [comment]
It's getting worse - sometimes there are 4 issues...

**Comment by kaelig** [comment]
Yeah, now I'm just using the Heroku Scheduler to run the bot every 10 minutes.
